mne/io/edf/tests/test_gdf.py

```diff
@@ -63,6 +64,8 @@ def test_gdf2_data():
     raw = read_raw_edf(gdf2_path + '.gdf', eog=None, misc=None, preload=True,
                        stim_channel='STATUS')
 
+    assert _handle_meas_date(raw.info['meas_date']) > 0
```

Comment:
This isn't necessary since it's in `_test_raw_reader` now, but it's okay to leave it if you want

Reply:
good catch. I forgot to take it out when rebaseing